scripts/zones/Halvung/npcs/_1qp.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Halvung
+--  NPC: Decorative Bronze Gate (_1qp)
+-----------------------------------
+local ID = require("scripts/zones/Halvung/IDs")
+require("scripts/globals/npc_util")
+require("scripts/globals/items")
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    if
+        npcUtil.tradeHasExactly(trade,
+        {
```

Comment:
Styling-wise, I can be on board with this, but I'd probably prefer breaking out the block on the `(` level

Reply:
Disregard this comment